contents/blog/running-group-targeted-ab-tests.md

```diff
@@ -0,0 +1,93 @@
+
+## How does running a group-targeted experiment differ from a user-targeted one?
+
+There are three key differences to take into account when running a group-targeted experiment versus a user-level one:
+
+### 1. Less statiscal power
+
+Since you treat a group of users as a single data point in group-targeted experiments, you have less statistical power. For example, Slack has 20 million users, but only 600,000 companies using it.
+
+In practice, this means that you'll usually have to run group-targeted experiments longer than you would user-targeted experiments. It also means that your results may be more susceptible to noise.
+
+### 2. Higher randomization risk
+
+Randomization risk can occur when groups are not properly randomized when assigned to the control or test variants. Since you have fewer data points in group-targeted experiments, the risk is higher that this will occur and it can distort results.
+
+For example, say a B2B SaaS company wants to see if a new pricing plan helps retain customers. They split companies into two groups: one with the new plan and one with the old plan. However, the randomization process assigns more small companies to the new plan and more large companies to the old plan. This may skew the results because company size could be a factor in determining customer retention rather than the pricing plan itself.
+
+### 3. Fewer user-level insights
+
+Since group-targeted experiments provide insights on an aggregate level, they may not show insights on a user level.
+
+For example, say a B2B SaaS tool wants users to invite coworkers to use the app. They add a big "Invite Coworker" button and test it in a company-targeted experiment. The results show an increase in invites sent, but it's hard to know how individual users used the button. For example, what are the roles or skill sets of team members who are most likely to interact with the button and send invites?
+
+## Combining user-targeted and group-targeted experiments to reduce risk
+
+To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:
+
+### 1. Save time for changes that don't have an impact. 
+
+As mentioned, lower statistical power in group-targeted experiments means you need to run your experiments longer. However, a change that does not impact behavior at a user-level will not likely translate into impact at a group-level. So you may waste time waiting for results from a group-level test that has no impact.
+
+In this case, it may be worth running a user-targeted experiment first in order to see if there is any impact on individual user behavior (which should be quicker to gain insights from because of higher statistical power) before running a group-targeted experiment.
```

Comment:
```suggestion
In this case, it may be worth running a user-targeted experiment first in order to see if there is any impact on individual user behavior (where it should be quicker to gain results from because of higher statistical power) before running a group-targeted experiment.
```